modules/virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume_{context}"]
+
+= Reclaiming persistent volumes for binding to a new persistent volume
+
+Use this procedure to reclaim a statically-provisioned persistent volume for reuse.
+
+.Prerequisite
+* Note the connection details of the persistent volume you want to reclaim.
+Use the Web Console or command line to display storage information for the
+virtual machine.
+
+.Procedure
+. Display the Storage Class using the Web Console or command line.
+Ensure that the Storage Class `reclaimPolicy` is set to `delete`.
+. Use the `oc describe pvc` command to verify that no other resources
```

Comment:
I think that you should present the command in a code block, like the rest of the commands.

Reply:
. Enter the following command to verify that no other resources
are using the persistent volume claim:
+
----
$ oc describe pvc <pvc_name>
----